components/NavigationBar.js

```diff
@@ -0,0 +1,124 @@
+import Link from "next/link";
+import styled from "styled-components";
+import React from "react";
+import { useRouter } from "next/router";
+
+const StyledFooter = styled.footer`
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 60px;
+  background-color: white;
+`;
+const NavigationList = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  justify-content: space-around;
+`;
+const IconListItem = styled.li`
+  background-color: ${(props) => props.bgColor || ""};
```

Comment:
When passing props to a styled component, you should prefix them with "$", otherwise you will get a message like this in your terminal:

> styled-components: it looks like an unknown prop "bgColor" is being sent through to the DOM, which will likely trigger a React console error. If you would like automatic filtering of unknown props, you can opt-into that behavior via `<StyleSheetManager shouldForwardProp={...}>` (connect an API like `@emotion/is-prop-valid`) or consider using transient props (`$` prefix for automatic filtering.)
Warning: React does not recognize the `bgColor` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `bgcolor` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

```suggestion
  background-color: ${(props) => props.$bgColor || ""};
```